auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -324,29 +305,22 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
                 f"best mean dice: {best_metric:.4f} at epoch {best_metric_epoch}"
             )
 
+        del inputs, labels, outputs
```

Comment:
Does this manual deletion that PyTorch is unable to release the GPU mem if they are allocated in another process?

Reply:
what do you mean here?

Reply:
@dongyang0122 I forgot what I was looking for. Please ignore it.